config.toml

```diff
@@ -11,7 +11,7 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 [[menu.main]]
     name = "Do I qualify?"
     weight = 1
-    url = "https://visafinder.tw/gold-card-qualification/"
+    url = "application-faq/qualifications/"
```

Comment:
@fifieldt shouldn't we use https://goldcard.nat.gov.tw/en/apply/step-1/ instead?

Reply:
This button is the most clicked on our website and should help the office website to rank better in more google keyword searches. 

cc @jliao-tw